### ISSUE TITLE
Generic.WhiteSpace.ScopeIndent incorrect error when multi-line IF condition mixed with HTML

### ISSUE THREAD

**Issue by robinJonsson** [squizlabs/PHP_CodeSniffer#2396]
Hi,

I get this error when I mix HTML blocks with multiple statements
`[phpcs] Line indented incorrectly; expected 2 tabs, found 3`

__Sample code:__
```PHP
<?php

if (true) {
	?>
	<div>
		<?php

		if (true) {
			?>
			<?php
		}

		if (true
			|| false) {
			?> <!-- error here -->
			<?php
		}

		?>
	</div>
	<?php
}

?>
```

__Screenshot:__
:x:

![skarmavbild 2019-02-01 kl 12 20 12](https://user-images.githubusercontent.com/189353/52120226-ecc7cb80-261b-11e9-9f77-5e52ef768cdc.png)

:white_check_mark:

![skarmavbild 2019-02-01 kl 12 20 38](https://user-images.githubusercontent.com/189353/52120347-4c25db80-261c-11e9-8848-4fb65efc5609.png)

![skarmavbild 2019-02-01 kl 12 20 56](https://user-images.githubusercontent.com/189353/52120361-57790700-261c-11e9-8e23-2a6a71521305.png)

__Version:__
PHP_CodeSniffer version 3.4.1 (stable) by Squiz (http://www.squiz.net) (Git clone 2019-02-01)

**Comment by gsherwood** [comment]
Thanks for reporting this. The fix will be in 3.4.1.
